app/controllers/users.js

```diff
@@ -0,0 +1,19 @@
+const logger = require('../logger');
+const userServices = require('../services/users');
+const { notFoundError } = require('../errors');
+
+exports.signUp = async (req, res, next) => {
+  try {
+    const { body } = req;
+    const userCreated = await userServices.signUpUser(body);
+
+    if (!userCreated && !userCreated.id) {
+      return notFoundError('User not registered');
```

Comment:
I think that to could use `next` or `throw` to send error when user created failed. I prefer to use `next` when error is in controller when error is in server or interactor I prefer use `throw`